website/markdown/posts/2020-06-17-version-1.11.0/post.mdx

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: Easily setup signing for your project with Tuist 1.11.0 Volare
+categories:
+  [
+    Tuist,
+    release,
+    swift,
+    project generation,
+    xcode,
+    apple,
+    '1.11.0',
+    'fastlane',
+    'automation',
+  ]
+excerpt: The new Tuist release 1.11.0 Volare introduces a signing feature to help you with maintaining and integrating your signing artifacts.
+author: fortmarek
+---
+
+Hi! 👋
+
+I am really excited to write this blog post as we have a great new feature for you to use -
+and this feature is signing 🔑
+
+So, let's dive right into it 🏊‍♂️
+
+## Signing
+
+Signing is one of the most complicated domains that iOS developers 
+have to deal with in Xcode. And although Apple has introduced automatic signing feature,
+most teams have decided not to use it as it creates a messy environment for their team.
+
+While for automating the process of generating provisioning profiles you'll have to wait as
```

Comment:
I think it's better to talk about what we don't have at the end and start presenting the feature talking about what we are bringing.